packages/sui-i18n/README.md

```diff
@@ -91,6 +91,47 @@ i18n.n(1000) //=> 1,000
 i18n.n(1000000) //=> 1,000,000
 ```
 
+### Add or modify translations dinamically
+
+You can use the method `addTranslations` to add new literals or modify existent ones given a key.
```

Comment:
```suggestion
You can use the method `addTranslations` to add new literals or modify existing ones given a key.
```

sorry about the grammar nazi inside me 😉

Reply:
don't worry, thats so good! :)